tools/chapel-py/chplcheck.py

```diff
@@ -155,12 +170,16 @@ def main():
 
     ctx = chapel.core.Context()
     ast = ctx.parse(args.filename)
+    rc = chapel.replace.ReplacementContext(args.filename)
```

Comment:
What is is the addition of `rc` for here?

Reply:
rc (might be better named eventually) computes line number etc. information, making it possible to retrieve the indentation of a node. Though, I guess the column info is sufficient for that....

Reply:
Ah I missed that you were using `rc.node_indent` in `check_misleading_indentation`. That was not immediately clear on a first glance since the globals are defined in main. I'd consider making `rc` a parameter to `report` or just using the column info for the indent.